indico/modules/events/logs/client/js/components/LogEntryModal.jsx

```diff
@@ -62,6 +64,18 @@ export default class LogEntryModal extends React.Component {
     nextEntry();
   }
 
+  relatedEntries() {
```

Comment:
`showRelatedEntries`?

Reply:
Yes, but then I would also rename the `nextEntry` and `prevEntry` to `showNextEntry` and `showPrevEntry`.